hazelcast/src/main/java/com/hazelcast/spi/impl/SlowOperationLog.java

```diff
@@ -208,8 +211,10 @@ public JsonObject toJson() {
         public void fromJson(JsonObject json) {
             id = getInt(json, "id");
             startedAt = getLong(json, "startedAt");
-            durationNanos = getLong(json, "duration");
             lastAccessNanos = System.nanoTime();
+
+            // write durationMs last to enforce a write barrier
+            durationMs = getInt(json, "durationMs");
```

Comment:
I had, but I will have to read them again ;)

Regarding `lastAccessNs`: It was originally part of the JSON, this is the reason it was volatile in the first place. It doesn't make sense anymore, I just was too stupid to change the logic accordingly. This should be done in this PR.